.github/workflows/make-release-commit.yml

```
@@ -41,13 +41,15 @@ jobs:
       with:
         part: ${{ inputs.part }}
     - name: Create tag
+      working-directory: rust
       run: |
-        export TAG="v${cargo metadata --no-deps --format-version 1 | jq '.packages[0].version}'"
+        export TAG="v$(cargo metadata --no-deps --format-version 1 | jq '.packages[0].version')"
```

I needed this to get it to work locally
```suggestion
        export TAG="v$(cargo metadata --no-deps --format-version 1 | jq '.packages[0].version' | xargs echo)"
```